oracle/blockoracle.go

```diff
@@ -64,7 +64,9 @@ func (bo *MinerBlockOracle) BlockEligible(layerID types.LayerID) (types.ATXID, [
 		return types.ATXID{}, nil, fmt.Errorf("cannot calc eligibility, not synced yet")
 	}
 	epochNumber := layerID.GetEpoch(bo.layersPerEpoch)
-	bo.log.Info("asked for eligibility for epoch %d (cached: %d)", epochNumber, bo.proofsEpoch)
+	bo.log.With().Info("asked for eligibility for epoch",
+		log.Uint64("epoch_number", uint64(epochNumber)),
```

Comment:
in this line you can use either - `epochNumber.Field()` or `log.EpochID(epochNumber)`. this is a standard we try to use every time we mention an epoch id to reduce the number of possible keys and to make it easier to understand what to use as a convention. 
It's a little tricky because this log has two epoch IDs so the second one would have to stay the same using the `log.Uint64` call. I would change it key to `cached_epoch`.

Reply:
thanks for the comment! while fixing this I tried to poke around the other log statements to try to make it consistent but wasn't sure if there would be confusion since there were 2 epoch numbers in this print statement. will update styling accordingly!